lib/base.js

```diff
@@ -656,8 +625,8 @@ var MemberElement = BaseElement.extend({
   toRefract: function() {
     return {
       element: this.element,
-      attributes: this.attributes,
-      meta: this.meta.toObject(),
+      attributes: this.attributes.toValue(),
```

Comment:
suggested: keep convention of specifying meta before attributes as you do elsewhere.